components/camel-sip/src/main/java/org/apache/camel/component/sip/listener/SipSubscriptionListener.java

```diff
@@ -34,48 +35,106 @@
 import org.apache.camel.CamelException;
 import org.apache.camel.Exchange;
 import org.apache.camel.ExchangePattern;
-import org.apache.camel.component.sip.SipSubscriber;
+import org.apache.camel.component.sip.SipConsumer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * SipListener which will listen for and process NOTIFY and MESSAGE requests events
+ */
 public class SipSubscriptionListener implements SipListener {
+
+    /**
+     * Logger for this class
+     */
     private static final Logger LOG = LoggerFactory.getLogger(SipSubscriptionListener.class);
-    private SipSubscriber sipSubscriber;
+
+    /**
+     * The camel consumer to give the incoming SIP messages
+     */
+    private SipConsumer sipConsumer;
+
+    /**
+     * The potential dialog between this retrieving SIP point and the sender
+     */
     private Dialog subscriberDialog;
+
+    /**
+     * The dialog for when the sender made one
+     */
     private Dialog forkedDialog;
 
-    public SipSubscriptionListener(SipSubscriber sipSubscriber) {
-        this.setSipSubscriber(sipSubscriber);
+    /**
+     * Create a SipListener listening for subscription events
+     * @param sipConsumer the camel consumer creating this listener
+     */
+    public SipSubscriptionListener(SipConsumer sipConsumer) {
+        this.setSipConsumer(sipConsumer);
     }
 
+    /**
+     * Sends the message body from a SIP request event along the camel route.
+     * @param response the message body from the SIP request
+     * @throws CamelException when dispatching the message goes wrong
+     */
     private void dispatchExchange(Object response) throws CamelException {
         LOG.debug("Consumer Dispatching the received notification along the route");
-        Exchange exchange = sipSubscriber.getEndpoint().createExchange(ExchangePattern.InOnly);
+        Exchange exchange = sipConsumer.getEndpoint().createExchange(ExchangePattern.InOnly);
         exchange.getIn().setBody(response);
-        try {
-            sipSubscriber.getProcessor().process(exchange);
-        } catch (Exception e) {
+
+        //try to send the message along
+        try
```

Comment:
@nikvaessen it looks like you moved the ```{``` to the next line, please use checkstyle to catch things like this, it is described here:
http://camel.apache.org/building.html